src/query.rs

```diff
@@ -12,13 +14,14 @@ use crate::identifier::KeyPath;
 use crate::serialize::{DatastoreSerializeError, Serialize};
 
 use google_datastore1::schemas::{
-    CompositeFilter, CompositeFilterOp, Filter, KindExpression, LookupRequest, LookupResponse,
+    CompositeFilter, CompositeFilterOp, Filter, Key, KindExpression, LookupRequest, LookupResponse,
     PropertyFilter, PropertyFilterOp, PropertyOrder, PropertyOrderDirection, PropertyReference,
     Query, QueryResultBatchMoreResults, ReadOptions, RunQueryRequest, RunQueryResponse,
 };
 
 const DEFAULT_PAGE_SIZE: i32 = 50;
 
+#[async_trait]
 pub trait DatastorersQueryable<E>
 where
     E: Kind
```

Comment:
Why does `DatastorersQueryable<E>`  require `E` to implement `DatastorersQueryable<E>` seems a bit recursive/redundant?

Reply:
That was a really good question!
looks kind of strange, any you know what, with some minor changes the requirement could be removed.
Included fix in this PR, see commit: 
https://github.com/et-alia/datastorers/pull/52/commits/4c6a04b4d6ab7a5ef82466314a7fa2de60c121cd